lib/rom/sql/relation/reading.rb

```diff
@@ -1102,6 +1102,32 @@ def __join__(type, other, join_cond = EMPTY_HASH, opts = EMPTY_HASH, &block)
             raise ArgumentError, "+other+ must be either a symbol or a relation, #{other.class} given"
           end
         end
+
+        # Coerces {ROM::SQL::Attribute} arguments into their
+        # canonical form
+        #
+        # Used by relation functions that accept attributes but don't
+        # want aliased attributes to generate invalid sql
+        #
+        # @api private
+        def canonicalize_args(args)
+          if args.is_a?(Array)
+            args.map {|arg|
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Space between { and | missing.](https://app.codacy.com/gh/rom-rb/rom-sql/pullRequest?prid=5107465)